tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "forceConsistentCasingInFileNames": true,
     "lib": ["ES2019"],
     "module": "CommonJS",
+    "skipLibCheck": true,
```

Comment:
Do we need this line? We can change lib to `ES2022` if some libraries' are missing.

Reply:
Yes, I agree, I mistook it as something else. Thanks!